### PR TITLE
fix: cleanup links and url fragments

### PR DESCRIPTION
**Pull request by aschmahmann**

### CODE REVIEW

File: data/spotlight5.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   
-  The ipfs.io public gateway is maintained by Interplanetary Shipyard on behalf of the IPFS Foundation. You can reach us at <a href="https://filecoinproject.slack.com/archives/C0573QZRETS" target="_blank">#waterworks-community</a> on Filecoin Slack, on GitHub at <a href"https://github.com/ipshipyard/waterworks-community" target="_blank">ipshipyard/waterworks-community</a>, and on the IPFS discussion forums at <a href="https://discuss.ipfs.tech/" target="_blank">discuss.ipfs.tech</a>.
+  The ipfs.io public gateway is maintained by Interplanetary Shipyard on behalf of the IPFS Foundation. You can reach us at <a href="https://filecoinproject.slack.com/archives/C0573QZRETS" target="_blank">#waterworks-community</a> on Filecoin Slack, on GitHub at <a href="https://github.com/ipshipyard/waterworks-community" target="_blank">ipshipyard/waterworks-community</a>, and on the IPFS discussion forums at <a href="https://discuss.ipfs.tech/" target="_blank">discuss.ipfs.tech</a>.
 button: 
   label: "Learn More"
-  link: "#fourth"
```
Comment (aschmahmann) [inline data/spotlight5.yml:13]: Do we want this to go back up? Also in the following of "Learn More" links the FAQ section is skipped, was that intentional @cewood?

File: data/faqs.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ items:
       There will always be a role for gateways, but we anticipate that role will shrink as direct
       IPFS support becomes more prevalent.
   - name: |
-      How does the gateway <a href=https://docs.ipfs.tech/reference/http/gateway/#trustless-verifiable-retrieval" target="_blank">ensure</a> whether the data they return matches the Content ID (CID) in the request? Can I access the gateways in a trustless way?
```
Comment (aschmahmann) [inline data/faqs.yml:32]: Links don't seem to work in titles here

File: data/spotlight5.yml (hunk 1 above)
Comment (cewood) [reply data/spotlight5.yml:13]: Yes, I suspect it's a quirk of the theme. Didn't want to waste an hour digging into it.

File: data/faqs.yml (hunk 2 above)
Comment (cewood) [reply data/faqs.yml:32]: Makes sense, it's a heading, so not surprised it isn't allowing links in there.

File: data/spotlight5.yml (hunk 1 above)
Comment (aschmahmann) [reply data/spotlight5.yml:13]: Ok, going to merge this for now so people can see it and then will follow up to try fixing it

**Pull request merged**